package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "yoast-test-helper",
+  "version": "1.0.2",
+  "description": "A plugin to aid in testing and developing the Yoast SEO plugin and its extensions.",
+  "repository": "https://github.com/Yoast/yoast-test-helper.git",
+  "author": "Yoast",
+  "license": "SEE LICENSE IN licence.txt",
```

Comment:
Shouldn't this be `license.txt`?

Reply:
npm documentation says "{ "license" : "SEE LICENSE IN <filename>" }": https://docs.npmjs.com/files/package.json

Reply:
I get that. What I'm saying is that the filename `licence.txt` is spelled incorrectly if you consider we use American spelling.